nvc/nvc.go

```diff
@@ -67,11 +68,6 @@ func Parse(r io.ReadSeeker) (Archive, error) {
 		order[i] = entry.Hash
 	}
 
-	// Sanity check
-	if len(entries) != len(order) {
-		panic(fmt.Sprintf("Lengths of Entries and EntryOrder do not match (%d vs %d)", len(entries), len(order)))
-	}
```

Comment:
are we assuming this invariant doesn't need testing?

Reply:
Since an `Archive` stores entries as a `map[Hash]TocEntry`, this caused a panic if a hash was duplicated in the table of contents